langlib/lang.future/src/main/ballerina/src/lang.future/future.bal

```diff
@@ -14,7 +14,9 @@
 // specific language governing permissions and limitations
 // under the License.
 
-# Cancels the given future.
-#
-# + f - The future to test
-public function cancel(future<any> f) = external;
+# Requests cancellation of a future.
+# + f - the future to be cancelled
+# This sets the cancellation flag in the strand corresponding to `f`.
```

Comment:
Shouldn't this bit come before the documentation for the parameter?

Reply:
Apparently this need to be fixed in the spec. Fixed it locally.